instructions_for_client

```diff
@@ -0,0 +1,21 @@
+Create an AWS t2.micro Instance according instructions
+
+In instance install Docker according instructions:
+https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-on-ubuntu-20-04
+
+Create directory for the project:
+$ mkdir apache_container
+$ cd apache_container
+
+Create folder with all necesary info for business card:
+$ mkdir business_card 
+
+Create Dockerfile
+$ cd ..                     (exit from busines card folder 1 level up)
+$ vim Dockerfile            (Apache image used from Dockerhub)
```

Comment:
Missing further instructions - like either use created Dockerfile for image building or what ?